content/python/concepts/built-in-functions/terms/setattr/setattr.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: 'Setattr()'
+Description: 'Sets the value of the attribute of an object.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`Setattr()`** function is a built-in Python function used to set the value of a named attribute of an object. It allows a developer to dynamically assign or modify attributes of an object at runtime.
+
+## Syntax
+
+```pseudo
+setattr(object, attribute, value)
+```
+
+The `setattr()` function requires three parameters:
+
+- `object`: an object.
+- `attribute`: attribute name that you want to set.
+- `value`: the value to give the `attribute`.
+
+## Example
+
+The following example will change the `value` of the `name` attribute from the `Person` object:
+
+```py
+class Person:
+  name = "Alex"
+  age = 30
+
+# Updating the name property
+setattr(Person, 'name', "John")
+
+# Retrieving the name property
+new_name = getattr(Person, 'name')
+print("My new name is " + new_name)
+
```

Comment:
```suggestion
setattr(Person, "name", "John")

# Retrieving the name property
new_name = getattr(Person, "name")
print("My new name is " + new_name)

```

Reply:
for the sake of consistency use single or double quotes exclusively within a code block. Also follow the code block with a sentence such as:

"This will result in the following output:"

And put the output in a shell block